coremltools/models/model.py

```diff
@@ -76,8 +76,14 @@ def _get_proxy_from_spec(filename):
         if _has_custom_layer(spec):
             # custom layers can't be supported directly by compiling and loading the model here
             return None
-
-        return _MLModelProxy(filename)
+        try:
+            return _MLModelProxy(filename)
+        except RuntimeError as e:
+            # print a warning of repr()
+            warnings.warn(
+                "Note: You will not be able to run predict() on this Core ML model", 
```

Comment:
I think we want to possibly show the actual message from `e`? Otherwise, it's not clear to the user why some models allow predict and some don't. Alternately, if we can actually determine what the various reasons are, we could craft messages specifically for each reason.

We also probably don't need the word "Note" in our warning (it already shows as a warning specifically, not just text output, right?).